docs.html

```diff
@@ -786,6 +792,69 @@ <h3>Modal Footer</h3>
 
 
 
+            <!-- Navigation Components -->
+            <section id="navigation">
+                <h2 class="component-heading">Navigation</h2>
+                <p class="component-desc">Basic Navigation for desktop</p>
+
+
+                <!-- The Navbar -->
+                <div class="navi">
+                    <h2>Title</h2>
+                    <ul>
+                        <li><a href="#">Item 1</a></li>
+                        <li><a href="#">Item 2</a></li>
+                        <li><a href="#">Item 3</a></li>
+                    </ul>
+                </div>
+
+                <pre></pre>
+                <iframe src="https://www.thiscodeworks.com/embed/62122e525362a500156c2b77"
+                    style="width: 100%; height: 359px;" frameborder="0"></iframe>
+                <hr>
+            </section>
+
+
+
+            <!-- stars Components -->
+            <section id="stars">
+                <h2 class="component-heading">Stars</h2>
+                <p class="component-desc">Basic stars for Ecommerce</p>
+
+
+                <!-- The stars -->
+                <div class="star-container ">
+                    <h4>
+                        Ratings:
+                    </h4>
+                    <label>
+                        <input class="star-input" type="checkbox" checked disabled>
+                        <i class="fas fa-star rating-star-icon"></i>
+                    </label>
+                    <label>
+                        <input class="star-input" type="checkbox" checked disabled>
+                        <i class="fas fa-star rating-star-icon"></i>
+                    </label>
+                    <label>
+                        <input class="star-input" type="checkbox" checked disabled>
+                        <i class="fas fa-star rating-star-icon"></i>
+                    </label>
+                    <label>
+                        <input class="star-input" type="checkbox" checked disabled>
+                        <i class="fas fa-star rating-star-icon"></i>
+                    </label>
+                    <label>
+                        <input class="star-input" type="checkbox" disabled>
+                        <i class="fas fa-star rating-star-icon"></i>
+                    </label>
+                </div>
+                <pre></pre>
```

Comment:
couldn't understand the use of pre tag here will you pls explain it

Reply:
The <pre> HTML element represents preformatted text which is to be presented exactly as written in the HTML file.